site/docs/v1/tech/identity-providers/samlv2-idp-initiated/okta.adoc

```diff
@@ -0,0 +1,131 @@
+
+This document assumes you have an admin account with Okta and a valid FusionAuth paid edition license. 
+
+index.adoc:include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+* <<Configure SAML v2 IdP Initiated SSO With Okta>>
+* <<Create and Partially Configure the Okta SSO Application>>
+* <<Add the Okta Public Certificate to FusionAuth>>
+* <<Add the SAMLv2 IdP Initiated Identity Provider>>
+* <<View the Identity Provider in FusionAuth>>
+* <<CORS Settings>>
+* <<Configure the FusionAuth Application Redirect URL>>
+* <<Complete SSO Configuration in Okta>>
+* <<Test It Out>>
+
+== Create and Partially Configure the Okta SSO Application
+
+Navigate to the Okta admin screen, and add an application. Search for `SAML Service Provider` and add this type of application.
+
+image::identity-providers/idp-initiated/okta-add-saml-service-provider-initial-screen.png[Setting up a SAMLv2 application in Okta,width=1200]
+
+Add an [field]#Application label# describing the application. Configure any other "General Settings" as needed.
+
+image::identity-providers/idp-initiated/okta-general-settings.png[General settings tab when creating a SAMLv2 application in Okta,width=1200,role=bottom-cropped]
+
+Click "Next". You will arrive at the "Sign-On Options" section.
+
+image::identity-providers/idp-initiated/okta-sign-in-options.png[Sign-on settings tab when creating a SAMLv2 application in Okta,width=1200]
+
+Click "View Setup Instructions". This will open needed instructions in a separate browser tab:
+
+image::identity-providers/idp-initiated/okta-saml-sp-configuration-instructions.png[Configuration instructions for the SAML SP when creating a SAMLv2 application in Okta,width=1200,role=bottom-cropped]
+
+Record the [field]#Identity Provider Issuer# and [field]#Identity Provider HTTP Post URL# values. The former is a string such as `exkq14ymac31Bx7895d6`. There may be a typo in the instructions with the string `is` prefixed to the [field]#Identity Provider Issuer#. 
+
+Download the "Identity Provider Certificate" too, then close the instructions tab.
+
+== Add the Okta Public Certificate to FusionAuth
+
+Log in to the FusionAuth administrative user interface and navigate to [breadcrumb]#Settings -> Key Master#. 
+
+Import the Okta provided certificate you just downloaded.
+
+image::identity-providers/idp-initiated/add-okta-public-certificate-to-keymaster.png[Adding the Okta certificate to Key Master,width=1200]
+
+== Add the SAMLv2 IdP Initiated Identity Provider
+
+Navigate to [breadcrumb]#Settings -> Identity Providers#. Add a SAMLv2 IdP Initiated Provider.
+
+* Configure the [field]#Name# with a descriptive value. 
+* Set the [field]#Issuer# value to the [field]#Identity Provider Issuer# value from Okta.
+* Set the [field]#Verification key# to the public certificate you just imported.
+
+Enable this Identity Provider for any FusionAuth applications; for this document, `Pied Piper` allows use of this Identity Provider. 
```

Comment:
consider 
Enable this Identity Provider for any FusionAuth applications you would like.  For this example, `Pied Piper` allows the use of this Identity Provider.